charts/squid/values.yaml

```diff
@@ -0,0 +1,3 @@
+squidImage: bmedora/squid-disk-cache:v0.0.1
+loadgenImage: ghcr.io/jsirianni/util:latest
```

Comment:
Lets build these images in this repo instead of relying on bmedora and jsirianni's github.